common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -328,6 +328,8 @@ def get_structure(self, key, course_context=None):
 
                 with TIMER.timer("get_structure.find_one", course_context) as tagger_find_one:
                     doc = self.structures.find_one({'_id': key})
+                    if doc is None:
```

Comment:
Leaving this breadcrumb here too - doc can be none when you're reading from a secondary that is experiencing replica lag and has not yet received this new structure.